lib/src/spec/realtime/realtime.dart

```diff
@@ -16,6 +16,6 @@ abstract class RealtimeInterface extends AblyBase {
   void close();
   void connect();
   final Connection connection;
-  RealtimeChannels channels;
+  C channels;
```

Comment:
I assume this should be `C get channels;` 
See also my comment in #33 `impl/rest/rest.dart`

Reply:
converted Channels and Connection to getters 9b2fc58aa8f37ae7d0e970599b689437ae58859e

from 33 => 68022946db52951e7223052bdfe67687d907b515